Classes/Domain/Model/RenderedNodeDomAddress.php

```diff
@@ -72,6 +72,26 @@ public function getFusionPath()
         return $this->fusionPath;
     }
 
+    /**
+     * Get the fusion path that should be used for rendering the addressed
+     * content. For most contents that would be the closest `Neos.Neos:ContentCase`
+     * within the path rather than the actual prototype that was used to
+     * render the content.
+     *
+     * @return string
+     */
+    public function getFusionPathForContentRendering()
+    {
+        $fusionPathForContentRendering = $this->getFusionPath();
+        $fusionPathForContentRendering = preg_replace(
+            '/(\/itemRenderer\<Neos\.\Neos\:ContentCase\>)\/default\<Neos\.Fusion\:Matcher\>\/element(\<.*\>)$/',
```

Comment:
The `\N` seems incorrect, but a few more escapes aren't required IMO
```suggestion
            '/(\/itemRenderer<Neos\.Neos:ContentCase>)\/default<Neos\.Fusion:Matcher>\/element(<.*>)$/',
```